src/py/flwr/simulation/app.py

```diff
@@ -70,6 +72,22 @@
 
 """
 
+NodeToPartitionMapping = Dict[int, int]
+
+
+def _create_node_id_to_partition_mapping(
+    partition_ids: List[str],
```

Comment:
Shouldn't these be integers?
```suggestion
    partition_ids: List[int],
```

Reply:
handled slightly different in latest commit